docs/customization.md

```diff
@@ -258,3 +274,24 @@ const MyComponent = () => (
   </ThemeConsumer>
 )
 ```
+
+---
+
+### Using the respective platform's native colors
+
+You may want to style your app using the native color palette. You can do this
+using the `colors` object and the `Platform` API.
+
+```jsx
+import { Platform } from 'react-native';
+import { colors } from 'react-native-elements';
+
+const theme = {
+  colors: {
+    Platform.select({
+      default: ...colors.platform.android,
+      ios: ...colors.platform.ios
+    })
+  }
+}
```

Comment:
Maybe right under this, add a simple example with Text element color

Reply:
Sure about that? I mean at this point the user has probably understood that all `primary` variables are overwritten.

Or do you mean an example showing how to apply the platform colors without spreading them like in this example?

Reply:
I added a simple example.

Reply:
Not like that, but using the color directly. 
```jsx
<Text style={{ color: theme.colors.platform.ios.sucess}}>Hey</Text>
```

Reply:
Done.

Reply:
@janhesters Sorry about that, now that i've read the entire document it looks a bit weird. Let's put back the example you had first

Reply:
Lol, no worries, okay 👍